manifests/deis-builder-rc.yaml

```diff
@@ -40,10 +58,6 @@ spec:
             - name: builder-key-auth
               mountPath: /var/run/secrets/api/auth
               readOnly: true
-            # not currently running minio with SSL support. see https://github.com/deis/minio/pull/22 for more detail
```

Comment:
Yup, it's removed entirely. [Here](https://github.com/arschles/deis-builder/blob/probes/manifests/deis-builder-rc.yaml) is the file on my branch, which is represented in the PR - notice no mention of SSL